### PR TITLE
[TD-14316]<docs>: add jdbc 2.0.38 doc

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

**Review by dingbo8128 — APPROVED** [review]
LGTM

File: documentation20/en/08.connector/01.java/docs.md [hunk 1]
```diff
@@ -55,15 +55,16 @@ INSERT INTO test.t1 USING test.weather (ts, temperature) TAGS('beijing') VALUES(
...
-| 2.0.37              |            X          |            X          | 2.4.0.6 以上           | 1.8.x    |
-| 2.0.36              |            X          | 2.2.2.11 以上          | 2.4.0.0 - 2.4.0.5     | 1.8.x    |
-| 2.0.35              |            X          | 2.2.2.11 以上          | 2.3.0.0 - 2.4.0.5     | 1.8.x    |
-| 2.0.33 - 2.0.34     | 2.0.3.0 以上           | 2.2.0.0 以上           | 2.4.0.0 - 2.4.0.5     | 1.8.x    |
-| 2.0.31 - 2.0.32     | 2.1.3.0 - 2.1.7.7     |            X          |            X          | 1.8.x    |
-| 2.0.22 - 2.0.30     | 2.0.18.0 - 2.1.2.1    |            X          |            X          | 1.8.x    |
-| 2.0.12 - 2.0.21     | 2.0.8.0 - 2.0.17.4    |            X          |            X          | 1.8.x    |
-| 2.0.4 - 2.0.11      | 2.0.0.0 - 2.0.7.3     |            X          |            X          | 1.8.x    |
+| ----------------------- | ------------------------ | ------------------------ | ------------------------ | ----------- |
+| 2.0.38                  | X                        | X                        | 2.4.0.12 以上            | 1.8.x       |
```
Comment (sangshuduo) [inline documentation20/en/08.connector/01.java/docs.md]: Chinese in English document

File: documentation20/en/08.connector/01.java/docs.md [hunk 2]
```diff
@@ -148,6 +149,16 @@ The JDBC-RESTful does not depend on the local function library. Compared with JD
 * JdbcUrl starts with "JDBC:TAOS-RS://"
 * Use port 6041 as the connection port
 
+Starting with version taos-jdbcdriver-2.0.38 and TDengine 2.4.0.12, **JDBC-RESTful** support batch load function. Data is transmitted between TAOS-JDBcDriver and TDengine through WebSocket connection. Compared with HTTP, WebSocket enables **JDBC-restful** to support large data volume query and improves query performance.
```
Comment (sangshuduo) [inline documentation20/en/08.connector/01.java/docs.md]: official feature name is bulk pulling

Comment (sangshuduo) [reply documentation20/en/08.connector/01.java/docs.md]: My version:
JDBC 2.0.38 and later version with TDengine 2.4.0.12 (and later version) start to support a new bulk-pulling feature. It provides higher data transmission performance and larger volume data queries capability via WebSocket communication.

File: documentation20/cn/08.connector/01.java/docs.md [hunk 3]
```diff
@@ -151,7 +152,18 @@ Connection conn = DriverManager.getConnection(jdbcUrl);
...
 3. 使用 6041 作为连接端口。
 
-如果希望获得更好的写入和查询性能，Java 应用可以使用 **JDBC-JNI** 的driver，如下所示：
+从 taos-jdbcdriver-2.0.38 和 TDengine 2.4.0.12 版本开始，**JDBC-RESTful** 的 driver 增加批量拉取数据功能。taos-jdbcdriver 与 TDengine 之间通过 WebSocket 连接进行数据传输。相较于 HTTP，WebSocket 可以使 **JDBC-RESTful** 支持大数据量查询，并提升查询性能。
+
+连接开启批量拉取方式：
+
+```
+String url = "jdbc:TAOS-RS://taosdemo.com:6041/?user=root&password=taosdata";Properties properties = new Properties();
+properties.setProperty(TSDBDriver.PROPERTY_KEY_BATCH_LOAD, "true");
```
Comment (sangshuduo) [inline documentation20/cn/08.connector/01.java/docs.md:161]: the official feature name is bulk pulling. what is batch load?

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see my comments

File: documentation20/en/08.connector/01.java/docs.md [hunk 4]
```diff
@@ -56,11 +56,11 @@ INSERT INTO test.t1 USING test.weather (ts, temperature) TAGS('beijing') VALUES(
 
 | taos-jdbcdriver version | TDengine 2.0.x.x version | TDengine 2.2.x.x version | TDengine 2.4.x.x version | JDK version |
 | ----------------------- | ------------------------ | ------------------------ | ------------------------ | ----------- |
-| 2.0.38                  | X                        | X                        | 2.4.0.12 以上            | 1.8.x       |
-| 2.0.37                  | X                        | X                        | 2.4.0.6 以上             | 1.8.x       |
-| 2.0.36                  | X                        | 2.2.2.11 以上            | 2.4.0.0 - 2.4.0.5        | 1.8.x       |
-| 2.0.35                  | X                        | 2.2.2.11 以上            | 2.3.0.0 - 2.4.0.5        | 1.8.x       |
-| 2.0.33 - 2.0.34         | 2.0.3.0 以上             | 2.2.0.0 以上             | 2.4.0.0 - 2.4.0.5        | 1.8.x       |
+| 2.0.38                  | X                        | X                        | 2.4.0.12 and above       | 1.8.x       |
```
Comment (sangshuduo) [inline documentation20/en/08.connector/01.java/docs.md]: and later is an idiom.

https://english.stackexchange.com/questions/112257/above-or-later-when-referencing-a-range-of-versions-of-software

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see my comments

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**